runconfig/parse.go

```diff
@@ -279,6 +279,11 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		return nil, nil, cmd, err
 	}
 
+	logConfig, err := parseLogConfig(*flLoggingDriver)
```

Comment:
Can we not actually parse here, but instead do this log at https://github.com/rancherio/docker/pull/2/files#diff-6ce9e79ddb91a3f06352abe1f2c72ecbR1388